generate.rb

```diff
@@ -8,98 +8,99 @@
 
 $env = ARGV[0]
 
-$assets = File.join('static', 'assets-web', '')
-$css =  File.join('static', 'css', '')
-$sass = File.join('static', 'sass', '')
-$javascript = File.join('static', 'javascript')
-$manifest = 'rev_manifest.json'
-$man_path = File.join('data', $manifest)
-$new_path = File.join($assets, 'app.css')
+#$path = File.join(".", "themes", "hugo-bootstrap")
```

Comment:
Most of the changes in this file are completely unnecessary. I cleared up the file (replaced " by ', removed interpolation when possible). Include only the relevant changes (the end of the file).